dapr/proto/runtime/v1/dapr.proto

```diff
@@ -60,6 +63,29 @@ message GetStateRequest {
   common.v1.StateOptions.StateConsistency consistency = 3;
 }
 
+// GetBulkStateRequest is the message to get a list of key-value states from specific state store.
+message GetBulkStateRequest {
+  // The name of state store.

```

Comment:
nit : can remove "The" from comment, though it is there already in other places..if possible might remove this unnecessary "The" .

Reply:
True, but it's so common on all other comments that I rather not create the inconsistency now..

Reply:
sounds good.